libs/ledger-live-common/.ci.bridge.env

```diff
@@ -0,0 +1,3 @@
+DISABLE_MOCKED_WARNING=true
+API_STELLAR_HORIZON_STATIC_FEE=true
```

Comment:
It is not directly related to the PR.
but it seems that the 
```
if (getEnv("API_STELLAR_HORIZON_STATIC_FEE")) {
```
in "src/families/stellar/api/horizon.ts"

could be replaced by something like
```
if (process.env.TESTING)
```
It could save one env var

Reply:
Ok thanks i'm going to check :)